js-envs/src/main/scala/org/scalajs/jsenv/phantomjs/PhantomJSEnv.scala

```diff
@@ -322,23 +322,18 @@ class PhantomJSEnv(
 
   protected trait AbstractPhantomRunner extends AbstractExtRunner {
 
+    protected[this] val codeCache = new VirtualFileMaterializer
```

Comment:
We have a binary incompatibility problem because of this:
```
[info] Scala.js JS Envs: found 2 potential binary incompatibilities while checking against /localhome/jenkins/scala-js-sbt-homes/sbt-home/local/org.scala-js/scalajs-js-envs_2.10/0.6.9/jars/scalajs-js-envs_2.10.jar 
[error]  * method codeCache()org.scalajs.jsenv.VirtualFileMaterializer in trait org.scalajs.jsenv.phantomjs.PhantomJSEnv#AbstractPhantomRunner does not have a correspondent in old version
[error]    filter with: ProblemFilters.exclude[MissingMethodProblem]("org.scalajs.jsenv.phantomjs.PhantomJSEnv#AbstractPhantomRunner.codeCache")
[error]  * synthetic method org$scalajs$jsenv$phantomjs$PhantomJSEnv$AbstractPhantomRunner$_setter_$codeCache_=(org.scalajs.jsenv.VirtualFileMaterializer)Unit in trait org.scalajs.jsenv.phantomjs.PhantomJSEnv#AbstractPhantomRunner does not have a correspondent in old version
[error]    filter with: ProblemFilters.exclude[MissingMethodProblem]("org.scalajs.jsenv.phantomjs.PhantomJSEnv#AbstractPhantomRunner.org$scalajs$jsenv$phantomjs$PhantomJSEnv$AbstractPhantomRunner$_setter_$codeCache_=")
```